plugins/modules/identity/ipa/ipa_pwpolicy.py

```diff
@@ -0,0 +1,247 @@
+description:
+- Add, modify, and delete password policy within IPA server.
+version_added: 1.3.0
+options:
+    cn:
+        description:
+        - Name of the group that the policy will apply to.
+        - If omitted, the global policy will be used.
+        aliases: ["group", "name"]
+        type: str
+    state:
+        description: State to ensure.
+        default: "present"
+        choices: ["absent", "present"]
+        type: str
+    krbmaxpwdlife:
+        description: Maximum password lifetime (in days).
+        aliases: ["maxlife"]
+        type: str
+    krbminpwdlife:
+        description: Minimum password lifetime (in hours).
+        aliases: ["minlife"]
+        type: str
+    krbpwdhistorylength:
+        description: Password history size.
+        aliases: ["history"]
+        type: str
+    krbpwdmindiffchars:
+        description: Minimum number of character classes.
+        aliases: ["minclasses"]
+        type: str
+    krbpwdminlength:
+        description: Minimum length of password.
+        aliases: ["minlength"]
+        type: str
+    cospriority:
+        description:
+        - Priority of the policy.
+        - High number means lower priority.
+        - Required when C(cn) is not the global policy.
+        aliases: ["priority"]
+        type: str
+    krbpwdmaxfailure:
+        description: Consecutive failures before lockout.
+        aliases: ["maxfail"]
+        type: str
+    krbpwdfailurecountinterval:
+        description: Period after which lockout is enforced (seconds).
+        aliases: ["failinterval"]
+        type: str
+    krbpwdlockoutduration:
+        description: Period for which lockout is enforced (seconds).
+        aliases: ["lockouttime"]
+        type: str
+extends_documentation_fragment:
+- community.general.ipa.documentation
+'''
+
+EXAMPLES = r'''
+- name: Modify the global password policy
+  community.general.ipa_pwpolicy:
+      maxlife: '90'
+      minlife: '1'
+      history: '8'
+      minclasses: '3'
+      minlength: '16'
+      maxfail: '6'
+      failinterval: '60'
+      lockouttime: '600'
+      ipa_host: ipa.example.com
+      ipa_user: admin
+      ipa_pass: topsecret
+
+- name: Ensure the password policy for the group admins is present
+  community.general.ipa_pwpolicy:
+      group: admins
+      state: present
+      maxlife: '60'
+      minlife: '24'
+      history: '16'
+      minclasses: '4'
+      priority: '10'
+      maxfail: '4'
+      failinterval: '600'
+      lockouttime: '1200'
+      ipa_host: ipa.example.com
+      ipa_user: admin
+      ipa_pass: topsecret
+
+- name: Ensure that the group sysops does not have a unique password policy
+  community.general.ipa_pwpolicy:
+      group: sysops
+      state: absent
+      ipa_host: ipa.example.com
+      ipa_user: admin
+      ipa_pass: topsecret
+'''
+
+RETURN = r'''
+pwpolicy:
+    description: Password policy as returned by IPA API
+    returned: always
+    type: dict
```

Comment:
A `sample:` value would be nice.

Reply:
Added in 92f88b266373cf6d9e4e71625b026776485e0d4e